simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/FilterableAccountListAdapter.kt

```diff
@@ -97,8 +97,8 @@ class AccountItemViewHolder(
   private val imageLoader: ImageLoaderType,
   private val onItemClicked: (AccountProviderDescription) -> Unit
 ) : RecyclerView.ViewHolder(itemView) {
-  private val accountIcon =
-    itemView.findViewById<ImageView>(R.id.accountIcon)
+  private val accountIcon: ImageView? =
+    itemView.findViewById(R.id.accountIcon)
```

Comment:
It looked like there's no longer an icon in the new library finder designs, so I made it nullable since some other layouts have it. It might make more sense just to have a separate ViewHolder now that you mention it

Reply:
Ah, yeah! Leonard did mention not delivering icons inline anymore. Maybe we only get the icon when we actually choose an account.

Reply:
We're leaving the icon off altogether in the finder, since most libraries don't have .SVG files. We might include them in other interfaces, though.